dom-event@^0.0.5 is unpublished and combokeys cannot be installed from NPM any more

```
> npm i combokeys
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/******/.nvm/versions/node/v5.7.1/bin/node" "/Users/******/.nvm/versions/node/v5.7.1/bin/npm" "i" "combokeys"
npm ERR! node v5.7.1
npm ERR! npm  v3.6.0

npm ERR! No compatible version found: dom-event@^0.0.5
npm ERR! Valid install targets:
npm ERR! 2.0.0
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/skladchikov/Documents/github/npm-debug.log
```

That's why it was unpublished https://medium.com/@azerbike/i-ve-just-liberated-my-modules-9045c06be67c#.f4lrbinzn

--- a/Combokeys/prototype/addEvents.js
+++ b/Combokeys/prototype/addEvents.js
@@ -2,7 +2,7 @@
 'use strict'
 module.exports = function () {
   var self = this
-  var on = require('dom-event')
+  var on = require('./dom-event')
   var element = self.element
 
   self.eventHandler = require('./handleKeyEvent').bind(self)

--- a/Combokeys/prototype/detach.js
+++ b/Combokeys/prototype/detach.js
@@ -1,4 +1,4 @@
-var off = require('dom-event').off
+var off = require('./dom-event').off
 module.exports = function () {
   var self = this
   var element = self.element

--- a/Combokeys/prototype/dom-event.js
+++ b/Combokeys/prototype/dom-event.js
@@ -1,0 +1,15 @@
+module.exports = on
+module.exports.on = on
+module.exports.off = off
+
+function on (element, event, callback, capture) {
+  !element.addEventListener && (event = 'on' + event)
+  ;(element.addEventListener || element.attachEvent).call(element, event, callback, capture)
+  return callback
+}
+
+function off (element, event, callback, capture) {
+  !element.removeEventListener && (event = 'on' + event)
+  ;(element.removeEventListener || element.detachEvent).call(element, event, callback, capture)
+  return callback
+}

--- a/package.json
+++ b/package.json
@@ -1,7 +1,4 @@
 {
-  "dependencies": {
-    "dom-event": "^0.0.5"
-  },
   "devDependencies": {
     "standard": "*",
     "browserify": "^9.0.3",

